src/middlewared/middlewared/plugins/reporting/arcstats.py

```diff
@@ -0,0 +1,44 @@
+class ZfsArcStats(object):
```

Comment:
Do we see any use case for this being a class ?

Reply:
Not really but it follows the same design as the `DiskStats` and `IfStats` classes to enforce some form of uniformity in this file 😄